Description: fix GCC-11 compilation
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-04-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Flawed/Makefile
+++ b/Flawed/Makefile
@@ -13,8 +13,8 @@
 DEFINES += -O2
 DEFINES_DBG += -ggdb -D_DEBUG
 
-CXXFLAGS += -Wall -pedantic -ansi -pipe $(shell $(PKG_CONFIG) --cflags sdl2 glu glew fmodex)
-CFLAGS += -Wall -pedantic -ansi -pipe $(shell $(PKG_CONFIG) --cflags sdl2 glu glew fmodex)
+CXXFLAGS += -Wall -pedantic -ansi -pipe $(shell $(PKG_CONFIG) --cflags sdl2 glu glew fmodex) -std=c++11
+CFLAGS += -Wall -pedantic -ansi -pipe $(shell $(PKG_CONFIG) --cflags sdl2 glu glew fmodex) -std=c++11
 LDFLAGS += $(shell $(PKG_CONFIG) --libs sdl2 glu glew fmodex)
 
 ifeq ($(DEBUG),yes)
